core/input/input.cpp

```diff
@@ -39,6 +39,87 @@
 #include "editor/editor_settings.h"
 #endif
 
+static const char *_joy_buttons[JOY_SDL_BUTTONS + 1] = {
+	"a",
+	"b",
+	"x",
+	"y",
+	"back",
+	"guide",
+	"start",
+	"leftstick",
+	"rightstick",
+	"leftshoulder",
+	"rightshoulder",
+	"dpup",
+	"dpdown",
+	"dpleft",
+	"dpright",
+	nullptr
+};
+
+static const char *_joy_button_names[JOY_BUTTON_MAX] = {
+	"Face Bottom",
```

Comment:
Out of curiosity, those "Face <direction>" names for the A, B, X and Y do not look very common to me. Where did you took this naming from ?

I fear it might confuse users.

Reply:
These were the names before:
https://github.com/godotengine/godot/blob/1176e078b965994964e7699efd41d9a8958db6da/core/input/input.cpp#L1319-L1349

Reply:
Oh yes, you are right, the diff confused me.
Thanks !